src/test/java/hw2/ex2/SecondScenarioTest.java

```diff
@@ -0,0 +1,278 @@
+        expectedServiceLeftSectionTexts.add("Table with pages");
+        expectedServiceLeftSectionTexts.add("Different elements");
+        expectedServiceLeftSectionTexts.add("Performance");
+        sa.assertEquals(actualServiceLeftSectionTexts,
+                expectedServiceLeftSectionTexts);
+
+        // 7. Open through the header menu Service ->
+        // Different Elements Page
+        // There is a simple opening of new
+        // page -- https://epam.github.io/JDI/different-elements.html
+        // and checking if the page is really opened
+        driver.findElement(By.xpath("//li[@class='dropdown']")).click();
+        driver.findElement(By.linkText("Different elements")).click();
+        assertEquals(driver.getCurrentUrl(),
+                "https://epam.github.io/JDI/different-elements.html");
+
+        // 8. Check interface on Different elements page,
+        // it contains all needed elements
+        // There we get all needable elements and check if there's
+        // really 4 checkboxes, 4 radios, 1 dropdown menu and 2
+        // different buttons
+        List<WebElement> checkButtons = driver.findElements(By.
+                className("label-checkbox"));
+        assertEquals(checkButtons.size(), 4);
+        List<WebElement> radioButtons = driver.findElements(By.
+                className("label-radio"));
+        assertEquals(radioButtons.size(), 4);
+        WebElement colorDropDown = driver.findElement(By.
+                className("colors"));
+        assertTrue(colorDropDown.isDisplayed());
+        WebElement defaultButton = driver.findElement(By.
+                xpath("//button[@name='Default Button']"));
+        assertTrue(defaultButton.isDisplayed());
+        defaultButton = driver.findElement(By.
+                xpath("//input[@class='uui-button']"));
+        assertTrue(defaultButton.isDisplayed());
+
+        // 9. Assert that there is Right Section
+        // There we check if the right section is displayed
+        assertTrue(driver.findElement(By.
+                xpath("//div[@name='navigation-sidebar']")).isDisplayed());
+
+        // 10. Assert that there is Left Section
+        // There we check if the left section is displayed
+        // NB! Soft Assert can be placed here 'cause we can
+        // continue test even if left section doesn't exist
+        // (we'll not use them in the further steps)
+        sa.assertTrue(driver.findElement(By.
+                xpath("//div[@name='log-sidebar']")).isDisplayed());
+
+        // 11. Select checkboxes
+        // There we should click on checkboxes with labels
+        // "Water" and "Wind" and check if they are checked
+        driver.findElement(By.
+                xpath("//div[@class='main-content']//div[2]//label[1]")).
+                click();
+        WebElement waterCheckBox = driver.findElement(By.
+                xpath("//div//div//div//div[2]//label[1]//input[1]"));
```

Comment:
This xpath does not readable. Please does not use auto generated xpath

Reply:
Solved: I create list of my checkboxes checked, if they are selected, by their numbers (we know what number has each checkbox, so I think it's a good idea). It's more simple to use and read now